src/components/notification/notification.module.scss

```diff
@@ -0,0 +1,68 @@
+@import 'src/assets/styles/variables/typography';
+@import 'src/assets/styles/mixins/font-scale';
+
+.notification-container {
+  display: flex;
+  box-sizing: border-box;
+  width: 600px;
```

Comment:
Should it be shorter on thinner screens?

Reply:
it was mentioned as fixed, but will check with the design team